ansible/centos-ci/ansible/roles/user/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+# Create a `ci` account with the following properties
+# - vagrant and libvirt group membership
+# - known ssh key in authorized_keys
+#
+# Also configure the `root` user to have a well known authorized_keys key
+
+- name: Create `ci` account
+  user:
+    name: ci
```

Comment:
Perhaps this should be namespaced as `leapp-ci` rather than using a generic account name?